File: 226_invert_binary_tree_lol/226.php
<?php // NOT NECESSARY, JUST FOR SYNTAX HIGHLIGHTING

class Solution {

    /**
     * @param TreeNode $root
     * @return TreeNode
     */
    function invertTree($root) {
        if ($root === null) {
            return null;
        }

        $temp = $root->left;
        $root->left = $root->right;
        $root->right = $temp;

        $this->invertTree($root->left);
        $this->invertTree($root->right);

        return $root;

    }
}

File: 226_invert_binary_tree_lol/226.cs
public class Solution {
    public TreeNode InvertTree(TreeNode root) {
        if (root == null)
            return null;

        TreeNode temp = root.left;
        root.left = root.right;
        root.right = temp;

        this.InvertTree(root.left);
        this.InvertTree(root.right);

        return root;
    }
}